cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaMirrorMaker2AssemblyOperatorTest.java

```diff
@@ -173,13 +197,20 @@ public void testCreateCluster(VertxTestContext context) {
                 assertThat(pdb.getMetadata().getName(), is(mirrorMaker2.getName()));
                 assertThat(pdb, is(mirrorMaker2.generatePodDisruptionBudget()));
 
-                // Verify status
                 List<KafkaMirrorMaker2> capturedMirrorMaker2s = mirrorMaker2Captor.getAllValues();
-                assertThat(capturedMirrorMaker2s.get(0).getStatus().getUrl(), is("http://foo-mirrormaker2-api.test.svc:8083"));
-                assertThat(capturedMirrorMaker2s.get(0).getStatus().getReplicas(), is(mirrorMaker2.getReplicas()));
-                assertThat(capturedMirrorMaker2s.get(0).getStatus().getLabelSelector(), is(mirrorMaker2.getSelectorLabels().toSelectorString()));
-                assertThat(capturedMirrorMaker2s.get(0).getStatus().getConditions().get(0).getStatus(), is("True"));
-                assertThat(capturedMirrorMaker2s.get(0).getStatus().getConditions().get(0).getType(), is("Ready"));
+
+                // Verify exclusions
+                KafkaMirrorMaker2MirrorSpec capturedMirrorConnector = capturedMirrorMaker2s.get(0).getSpec().getMirrors().get(0);
+                assertThat(capturedMirrorConnector.getTopicsExcludePattern(), is("other-topic"));
+                assertThat(capturedMirrorConnector.getGroupsExcludePattern(), is("other-group"));
```

Comment:
Do we have some test covering that the old API fields still work as well?

Reply:
Let me add one